7.6293945e-06 1.04904175e-05
0.017216528 20.294428
Traceback (most recent call last):
  File "/home/ws/fq0795/git/gnn_uncertainty/reevaluate_experiments.py", line 172, in <module>
    ens.evaluate_all(
  File "/home/ws/fq0795/git/gnn_uncertainty/uncertainty/base_uncertainty.py", line 388, in evaluate_all
    self._scatter_plot(
  File "/home/ws/fq0795/git/gnn_uncertainty/uncertainty/base_uncertainty.py", line 663, in _scatter_plot
    plt.xlim(x_min, x_max)
             ^^^^^
UnboundLocalError: cannot access local variable 'x_min' where it is not associated with a value
